WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-ColHeader {
+	border-bottom:0px;
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskListViewItem@
+}
+
+.ZmTaskMultiView .DwtListView .RowDouble td {
+	@TaskListViewItemCheckbox@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskSubject {
+	@TaskListViewItemSubject@
 }
 
-.newTaskBannerSep {
-	border-bottom:2px solid #AAA;
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskDate {
+	@TaskListViewItemDate@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow {
+	@TaskListViewItemBottomContainer@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconOuterWrapper {
+	@TaskListViewItemBottomIconOuterWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconInnerWrapper {
+	@TaskListViewItemBottomIconInnerWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-Rows  .ImgCheckboxUnchecked-svg {
+	padding-top:32px;
+}
+
+.ZmTaskMultiView .DwtListView  .Row-selected .ImgCheckboxChecked-svg {
+	padding-top:32px;
```

Comment:
The two different images thing sound ok to me now. 
About the height, I brought that up just because aligning vertically using padding is not the best way and in upcoming sprints we are going to have a circular progress bar which would most likely break this. I am fine with it if you think this won't create problem.